tests/app/main/test_briefs.py

```diff
@@ -1945,15 +1960,6 @@ def test_view_response_result_submitted_unsuccessful(self, data_api_client):
         doc = html.fromstring(res.get_data(as_text=True))
         assert doc.xpath('//h1')[0].text.strip() == "You don’t meet all the essential requirements"
 
-    def test_view_response_result_not_submitted_redirect_to_submit_page(self, data_api_client):
-        data_api_client.get_brief.return_value = self.brief
-        data_api_client.is_supplier_eligible_for_brief.return_value = True
-        data_api_client.find_brief_responses.return_value = {"briefResponses": []}
-        res = self.client.get('/suppliers/opportunities/1234/responses/result')
-
-        assert res.status_code == 302
-        assert res.location == 'http://localhost/suppliers/opportunities/1234/responses/create'
-
     def test_essential_skills_shown_with_response(self, data_api_client):
```

Comment:
No and I think this is the same issue mentioned in https://www.pivotaltracker.com:/story/show/130019135/comments/161850371. I'm gonna take it as such so the fix for that should cover this one too.